tests/Scoper/PhpScoperTest.php

```diff
@@ -965,5 +965,118 @@ function foo($bar) : Closure
 
 PHP
         ];
+
+        //
+        // Function parameters
+        //
+        // ====================
+
+        yield '[Function parameter] class_exists' => [
+            <<<'PHP'
+<?php
+
+class_exists('Symfony\\Component\\Yaml\\Yaml');
+class_exists('\\Symfony\\Component\\Yaml\\Yaml');
+
+PHP
+            ,
+            'Humbug',
+            <<<'PHP'
+<?php
+
+class_exists('Humbug\\Symfony\\Component\\Yaml\\Yaml');
+class_exists('Humbug\\Symfony\\Component\\Yaml\\Yaml');
```

Comment:
eeh, I would say both are correct so for the sake of consistency we could transform it to `'\\Humbug\\Symfony\\Component\\Yaml\\Yaml'` to be clearer